ddpui/api/user_org_api.py

```
@@ -9,8 +9,10 @@
 from django.db.models import Prefetch
 from django.contrib.auth.models import User
 from django.db.models import F
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
```

Ruff (0.11.9)

`rest_framework_simplejwt.serializers.TokenObtainPairSerializer` imported but unused

Remove unused import: `rest_framework_simplejwt.serializers.TokenObtainPairSerializer`

(F401)